make_ghpages/make_pages.py

```diff
@@ -79,6 +79,10 @@ def get_git_branches(git_url):
     return res
 
 def validate_meta_info(app_name, meta_info):
+    if not meta_info:
```

Comment:
Is this really what we want?
If no meta info is given, we want the build to fail, right?

Reply:
I don't know. And I still can't decide on it properly, I must admit.

On one side, it is the author's fault that false or no extra details will be provided.
But on the other side, it will look bad on us, if some apps are not properly presented.

Reply:
> it will look bad on us, if some apps are not properly presented.

The idea should be: if someone makes a PR with an app that is missing info, the build should fail - so that he or she will notice and fix the problem.

Reply:
Done.
Now checking that:
- Keys for each app entry in `apps.json` exists and is valid (except for 'categories')
- Values for each of those keys in `apps.json` is valid (reachable URLs and if category, it is checked that the supplied categoriy(-ies) exist(s) in `categories.json`

Raising if any of these things occur, printing a warning if 'categories' is not specified.